__tests__/components/checkout/payment-method-header.spec.js

```diff
@@ -5,13 +5,19 @@ import React from 'react'
 import PaymentMethodHeader from '../../../src/components/checkout/payment-method-header'
 
 test('renders the navigation button when collapsed', () => {
```

Comment:
id add 'and showEditButton is true' to this test as it can be collapsed but if you pass showEditButton false to it then the button wouldn't render